hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationConstants.java

```diff
@@ -90,6 +90,14 @@
     public static final int JAVA_DEFAULT_TYPE_EXTERNALIZABLE = -101;
 
     // ------------------------------------------------------------
+    // LANGUAGE SPECIFIC SERIALIZERS
+    // USED BY CLIENTS (Not deserialized by server)
+
+    public static final int CSHARP_CLR_SERIALIZATION_TYPE = -110;
```

Comment:
I do not know if they needed to increase by 10. However these numbers are already in use. We put them here to prevent somebody picking up these numbers for a different purpose in future.